app/pods/cases/case/subcases/subcase/controller.js

```diff
@@ -1,9 +1,12 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
 
 export default class CasesCaseSubcasesSubcaseController extends Controller {
+  @service router;
+
   @action
   refreshSubcases() {
-    this.send('refreshParentModel');
+    this.router.refresh('cases.case.subcases');
```

Comment:
this only made sense when the subcases were in the model of `cases.case.subcases`, if we keep the current approach, you would just need to refresh the current route `cases.case.subcases.subcase`